pose0:
  x: 0.35
  y: 0
  z: 0.2
  X: 0
  Y: 0
  Z: 0
pose1:
  x: 0.122
  y: 0
  z: 0
  X: 0
  Y: 0
  Z: 0
pose2:
  x: 0
  y: 0
  z: 0.1
  X: 0
  Y: -0.785
  Z: 0